argilla-server/CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ These are the section headers that we use:
 
 ## [Unreleased]()
 
+## Added
```

Comment:
The changelog entry should include more context about the configuration switch to help users understand its purpose.